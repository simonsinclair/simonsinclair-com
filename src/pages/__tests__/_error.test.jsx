/* eslint-env jest */

import { render } from '@testing-library/react';

import Error from '../_error';

// To do: learn to mock out nested components properly.
jest.mock('../../components/Page', () => 'Page');

describe('<Error />', () => {
  it('Should match Snapshot', () => {
    // asFragment - Returns a DocumentFragment of your rendered component.
    const { asFragment } = render(<Error />);
    expect(asFragment()).toMatchSnapshot();
  });
});
