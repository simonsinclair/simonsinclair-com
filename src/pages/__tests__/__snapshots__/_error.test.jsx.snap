// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> Should match Snapshot 1`] = `
<DocumentFragment>
  <page
    title="Simon Sinclair â€” Page Not Found"
  >
    <main
      class="container"
    >
      <h1>
        Page Not Found
      </h1>
      <p>
        Sorry, the page you're trying to view cannot be found.
      </p>
    </main>
  </page>
</DocumentFragment>
`;
